pages/doc/wavefront_spring_boot.md

```diff
@@ -512,47 +444,14 @@ You can configure the cluster and shard the same way. This information is used t
 * If you want to take full control over [`ApplicationTags`](trace_data_details.html#application-tags), you can create a `@Bean`.
 * If you want to customize the instance that is auto-configured, add an `ApplicationTagsBuilderCustomizer` bean.
 
-<!---
-<p><span style="font-size: large; font-weight: 600">Disable Integration Tests from Sending data to Wavefront</span></p>
-
-Add `management.metrics.export.wavefront.enabled` to your integration tests and set it to `false` to stop it from sending data to Wavefront.
-
-{% include note.html content="If you use the [Spring Initializer](https://start.spring.io/) to create your new Spring Boot application and add the Wavefront dependency, this configuration is added by default to all your integration tests."%}
-
-Example: Disable all integration tests from sending data to Wavefront by updating the `application.properties` file.
-```
-management.metrics.export.wavefront.enabled=false
-```
-
-Example: Disable a specific integration test from sending data to Wavefront.
-```
-  package com.example.demo;
-
-  import org.junit.jupiter.api.Test;
-  import org.springframework.boot.test.context.SpringBootTest;
-  import org.springframework.test.context.TestPropertySource;
-
-  @TestPropertySource(properties = "management.metrics.export.wavefront.enabled=false")
-  @SpringBootTest
-  class DemoApplicationTests {
-
-  	@Test
-  	void contextLoads() {
-  	}
-
-  }
-  ```
-
-{% include note.html content="For details on customizing exported metrics, check the [Spring Boot reference guide](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-metrics-export-wavefront)."%}--->
 
 ## Wavefront Spring Boot Integration
 
-If you already have a Wavefront account, you can start the setup and examine the dashboards from the Wavefront Spring Boot integration. This is supported for [Wavefront trial accounts](https://tanzu.vmware.com/observability).
+If you already have a Operations for Applications account, you can start the setup and examine the dashboards from the Wavefront Spring Boot integration. This is supported for [free trial accounts](https://tanzu.vmware.com/observability).
 
-1. Log in to your Wavefront instance.
-2. Select **Integrations**, search for Spring Boot, and select the Spring Boot integration.
-3. Use the information in the **Setup** tab to set up the integration.
-4. When setup is complete, click the **Dashboard** tab to examine your data.
+1. Select **Integrations**, search for Spring Boot, and select the Spring Boot integration.
```

Comment:
Click **Integrations** on the toolbar, search for Spring Boot, and click the Spring Boot integration.